_articles/github-for-collaboration/collaborative-workflow-for-contributions.md

```diff
@@ -60,6 +60,10 @@ When you’re writing commit messages, you’ll want to make them short, clear,
     4.  Write a great commit message
     5.  Make a pull request!
 
+![New Pull Request]({{ site.baseurl }}/img/create-pull-req-1.png)
```

Comment:
Is there some markdown magic/html snippet (like for the videos) to get a "caption"? Otherwise just having the text would work too.